README.md

```diff
@@ -1 +1,200 @@
-# volttron-emailer
\ No newline at end of file
+volttron-emailer
+=======
+
+The Emailer agent allows an instance of the VOLTTRON platform to send
+email.  When used in combination with the Alert agent, alerts from
+unpublished configured devices will automatically be sent.  In addition,
+agents are able to send emails directly through the pub/sub interface.
+
+Agents needing to send an email through the instance can do so by
+sending the following header and message to the `platform/send_email` topic
+which is monitored by the Emailer agent.  The following 
+is the expected payload for the message body and the optional header.
+
+
+# Prerequisites
+
+* Python 3.8
```

Comment:
So this requires python 3.10 as volttron requires 3.10 for modular.

Please modify